eng/common/scripts/Generate-DocIndex.ps1

```diff
@@ -0,0 +1,147 @@
+        if ($packageNameRegex) {
+            $artifact = $artifact -replace $packageNameRegex, $packageNameReplacement
+        }
+        # Read the artifact package infomation from csv of Azure/azure-sdk/_data/release/latest repo.
+        $packagesInfo = $metadata | ? { $_.Package -eq $artifact -and ($_.GroupId -eq $defaultGroupId)}
+        if ($packagesInfo.length -lt 1) {
+            $packagesInfo = $metadata | ? { $_.Package -eq $artifact }
+        }
+        if (!$packagesInfo) {
+            Write-Error "Did not find the artifacts from release csv. Please check and update."
+        }
+        elseif ($packagesInfo.length -gt 1) {
+            Write-Warning "There are multiple artifacts fetching out for the same package name. "
+            Write-Warning "Please check csv of Azure/azure-sdk/_data/release/latest repo if this is intended. "
+        }
+        foreach ($pkg in $packagesInfo) {
+            # Ignore the one marked as Hide
+            $hidden = $pkg.Hide
+            if ($hidden -and $hidden.Trim() -eq "true") {
+                continue
+            }
+            $serviceName = $pkg.ServiceName
+            # If no service name retrieved, print out warning message, and put it into Other page.
+            if (!$serviceName) {
+                Write-Warning "Please double check and update the artifact correct service name to corresponding repo: https://github.com/Azure/azure-sdk/tree/master/_data/releases/latest."
+                Write-Warning "If the package is not relavant, please set Hide value to true."
+                $serviceName = "Other"
+            }
+            $serviceName = $serviceName.Trim()
+            # The name of generating md files.
+            $fileName = ($serviceName -replace '\s', '').ToLower().Trim()
+            if ($orderSet.ContainsKey($fileName)) {
+                Add-Content -Path "$($YmlPath)/${fileName}.md" -Value "#### $artifact"
+            }
+            else {
+                New-Item -Path $YmlPath -Name "${fileName}.md" -Force
+                Add-Content -Path "$($YmlPath)/${fileName}.md" -Value "#### $artifact"
+                $orderSet[$fileName] = $serviceName
+            }
+        }
+    }
+    # Fetch page token
+    $pageToken = $xmlDoc.EnumerationResults.NextMarker
+} while ($pageToken)
+
+# Sort and display toc service name by alphabetical order.
+foreach ($serviceMapping in $orderSet.getEnumerator() | Sort Key) {
+    $fileName = $serviceMapping.Key
+    $serviceName = $serviceMapping.Value
+    Add-Content -Path "$($YmlPath)/toc.yml" -Value "- name: ${serviceName}`r`n  href: ${fileName}.md"
+}
+
+Write-Verbose "Creating Site Title and Navigation..."
+New-Item -Path "${DocOutDir}" -Name "toc.yml" -Force
+Add-Content -Path "${DocOutDir}/toc.yml" -Value "- name: Azure SDK for $titleRegex APIs`r`n  href: api/`r`n  homepage: api/index.md"
+
+Write-Verbose "Copying root markdowns"
+Copy-Item "$($RepoRoot)/README.md" -Destination "${DocOutDir}/api/index.md" -Force
```

Comment:
One difference that I have seen across the languages is some have "/api/" in the path and some don't. This will make them consistent but I'd also recommend that we remove "/api/" from the path as it doesn't seem necessary.